sdcm/nemesis.py

```diff
@@ -48,6 +48,7 @@
     NodeSetupFailed,
     NodeSetupTimeout,
     NodeStayInClusterAfterDecommission,
+    MAX_TIME_WAIT_FOR_NEW_NODE_UP,
```

Comment:
Please, place it in alphabetical order - between D and N.

Reply:
done